samcli/lib/remote_invoke/exceptions.py

```diff
@@ -1,10 +1,17 @@
 """
 Exceptions used by remote invoke executors
 """
-from samcli.commands.exceptions import UserException
 
 
-class InvalidResourceBotoParameterException(UserException):
+class InvalidResourceBotoParameterException(Exception):
+    msg: str
+
+    def __init__(self, msg: str) -> None:
+        self.msg = msg
+        super().__init__(self.msg)
```

Comment:
nit: I think we don't need to have this part if you just want to keep `msg` field there.